package.json

```
@@ -104,11 +104,11 @@
     "typescript": "4.1.5"
   },
   "peerDependencies": {
-    "@types/react-relay": ">=7.0.0",
-    "@types/relay-runtime": ">=6.0.7",
-    "relay-compiler": ">=9.0.0",
-    "relay-runtime": ">=9.0.0",
-    "typescript": ">=3.6.4"
+    "@types/react-relay": ">=7.0.17",
+    "@types/relay-runtime": ">=10.1.8",
+    "relay-compiler": ">=10.1.3",
+    "relay-runtime": ">=10.1.3",
+    "typescript": ">=4.1.5"
```

I just bumped to the latest version. SInce typescript does not really follow semantic versioning we would have to verify each minor version manually and then whitelist it or create a new major release?
```suggestion
    "typescript": ">=4.1.5 < 4.2.0"
```

---

I don't think being too strict is a good thing

---

Then we should add a statement to the readme that the library tries to follow semver as close as possible aside from breaking changes introduced by typescript?

---

I added a small notice in the install section.